src/app/modules/navbar/navbar.component.html

```diff
@@ -0,0 +1,70 @@
+<div class="navbar-index" *ngIf ="currentUrl !== 'auth'" [ngClass]="currentUrl == 'landing' ? 'blue-nav' : 'white-nav'">
+  <nav class="navbar-space row no-gutters">
```

Comment:
@Nishanthh I think the navbar should be transparent for landing instead of setting it to blue